test/testsimplifytemplate.cpp

```diff
@@ -1398,6 +1401,8 @@ class TestSimplifyTemplate : public TestFixture {
                       "template<class T> unsigned A<T>::foo() { return 0; }", 19));
         ASSERT_EQUALS(9, templateNamePositionHelper("template<class T, class U> class A { unsigned foo(); }; "
                       "template<class T, class U> unsigned A<T, U>::foo() { return 0; }", 25));
+        ASSERT_EQUALS(9, templateNamePositionHelper("template<class T, class U> class A { unsigned foo(); }; "
```

Comment:
Good to see unit testing :-)
Should one try more complex things like template template parameters?